homeassistant/components/template/light.py

```diff
@@ -613,6 +508,10 @@ async def async_turn_on(self, **kwargs: Any) -> None:  # noqa: C901
                 self._rgbw_color = None
             optimistic_set = True
 
+        return optimistic_set
+
+    def run_registered_scripts(self, **kwargs) -> tuple[str, dict]:
```

Comment:
I suggest to change the name like this because:
- The function doesn't execute script, it returns the script
- The function returns a single script, not several
```suggestion
    def get_registered_script(self, **kwargs) -> tuple[str, dict]:
```